pilot/prompts/development/implement_changes.prompt

```diff
@@ -16,7 +16,7 @@ This task is broken down into steps. Please tell me the changes needed to implem
 {{ step_description }}
 ```
 
-Within the file modifications, anything needs to be written by the user, add the comment in the same line as the code that starts with `// INPUT_REQUIRED {input_description}` where `input_description` is a description of what needs to be added here by the user. Finally, you can save the modified files on the disk by calling `save_files` function.
+Within the file modifications, anything needs to be written by the user, add the comment in the same line as the code that starts with `// INPUT_REQUIRED {input_description}` where `input_description` is a description of what needs to be added here by the user. Just make sure that you put comments only inside files that support comments (e.g. not in JSON files)..
```

Comment:
Will that actually fix this problem?

```
/* INPUT_REQUIRED {Download Bootstrap minified CSS from [getbootstrap.com](http://getbootstrap.com/) or use a CDN link to copy its content here} */
```

I got that in one of my tests. Comments are allowed in `.js` files but the point here is to have it filled in, and it wasn't. I don't think this change will solve that?

Reply:
The problem you're referring to is made by adding the instructions to add INPUT_REQUIRED to parse_task.prompt that didn't have it before and since it's used in the beginning of the project, it added the comment to .json file so this patch makes sense